addons/mail/static/src/js/thread.js

```diff
@@ -74,6 +75,7 @@ var Thread = Widget.extend({
             squash_close_messages: true,
             display_email_icon: true,
             display_reply_icon: false,
+            is_touch_screen: config.device.touch
```

Comment:
This should not be done like this. In the enterprise repo, the `o_touch_device` class is automatically set on the `.o_web_client`. If we need to use this detection in community, the enterprise code that does that should be moved to community (web_enterprise -> web) so that no JS modification would be needed here.